VotingApplication/VotingApplication.Web.Api/Models/DBViewModels/VoteRequestModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace VotingApplication.Web.Api.Models.DBViewModels
+{
+    public class VoteRequestModel
+    {
+        [Required]
+        public long OptionId { get; set; }
+
+        [Required]
+        public Guid TokenGuid { get; set; }
+
+        [Range(0, int.MaxValue)]
```

Comment:
The basic vote sends votes with a 0 VoteValue. Changing this to +1 would break that.